boards/seeed/xiao_mg24/xiao_mg24.dts

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2025 Pete Johanson
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <silabs/efr32mg24b220f1536im48.dtsi>
+#include "xiao_mg24-pinctrl.dtsi"
+#include "seeed_xiao_connector.dtsi"
+
+/ {
+	model = "Seeed XIAO MG24";
+	compatible = "seeed,xiao_mg24", "silabs,efr32mg24";
+
+	chosen {
+		zephyr,console = &usart0;
+		zephyr,shell-uart = &usart0;
+		zephyr,uart-pipe = &usart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,bt-hci = &bt_hci_silabs;
+	};
+
+	/* These aliases are provided for compatibility with samples */
+	aliases {
+		led0 = &led0;
+		watchdog0 = &wdog0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpioa 7 GPIO_ACTIVE_LOW>;
+			label = "LED 0";
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <78000000>;
+};
+
+&pstate_em3 {
+	status = "disabled";
+};
+
+&hfxo {
+	status = "okay";
+	ctune = <95>;
+	precision = <50>;
+};
+
+&lfxo {
+	status = "okay";
+	ctune = <44>;
+	precision = <50>;
+};
+
+&hfrcodpll {
+	clock-frequency = <DT_FREQ_M(78)>;
+	clocks = <&hfxo>;
+	dpll-n = <3839>;
+	dpll-m = <1919>;
+	dpll-edge = "fall";
+	dpll-lock = "phase";
+	dpll-autorecover;
+};
+
+&em23grpaclk {
+	clocks = <&lfxo>;
+};
+
+&em4grpaclk {
+	clocks = <&lfxo>;
+};
+
+&sysrtcclk {
+	clocks = <&lfxo>;
+};
+
+&wdog0clk {
+	clocks = <&lfxo>;
+};
+
+&wdog1clk {
+	clocks = <&lfxo>;
+};
+
+&usart0 {
+	current-speed = <115200>;
```

Comment:
USART0 is used to communicate with the debugger. The debugger seems to require 9600 baud for the data to come through correctly on the USB-CDC endpoint.

Reply:
Odd... This *fails* here when set to 9600, and works fine as is for me here locally, using `tio` on Linux to connect to the CDC ACM device. Did you test with 9600 locally and have it work as expected?

Reply:
After retesting, it seems like the CMSIS-DAP debugger respects USB-CDC control signals to use whatever baudrate the terminal requests, so I'm fine with leaving it at 115200 by default.